core/builder/src/main/java/io/quarkus/builder/BuildChainBuilder.java

```diff
@@ -186,10 +187,12 @@ public BuildChain build() throws ChainBuildException {
                         if (produce.getConstraint() == Constraint.REAL
                                 && produce.isOverridable() == overridable) {
                             final Throwable cause = new Throwable("This is the location of the conflicting producer ("
-                                    + toBeAdded.getStepBuilder().getBuildStep() + ")");
+                                    + toBeAdded.getStepBuilder().getBuildStep()
+                                    + ").  Use -Dquarkus.builder.log-conflict-cause=true to see the full stacktrace.");
```

Comment:
Isn' t this message being displayed if `-Dquarkus.builder.log-conflict-cause=true` is set already?

Reply:
I changed it so that this line is included conditionally based on LOG_CONFLICT_CAUSING.